src/pystatis/table.py

```diff
@@ -142,6 +148,19 @@ def get_data(
             "format": "ffcsv",
         }
 
+        if classifyingvariable1:
```

Comment:
you actually don't need to check the variable here, it doesn't hurt to add empty variables this will be just ignored by the API